root/static/scripts/common/components/TagEditor.js

```diff
@@ -290,7 +290,9 @@ class TagEditor extends React.Component<TagEditorProps, TagEditorState> {
       };
 
       if (!this.state.positiveTagsOnly || isAlwaysVisible(t)) {
-        const isGenre = this.genreMap.hasOwnProperty(t.tag.name);
+        const isGenre = Object.prototype.hasOwnProperty.call(
+          this.genreMap, t.tag.name
+        );
```

Comment:
This should change into two lines in the same way as the Autocomplete.js one :)

Reply:
Sounds good. Thanks so much and happy holidays!